spec.emu

```diff
@@ -62,7 +62,7 @@ contributors:
             1. Return the string-concatenation of 0x005C (REVERSE SOLIDUS) and the string in the "ControlEscape" column of the row whose "Code Point" column contains _c_.
           1. Let _otherPunctuators_ be the string-concatenation of *",-=<>#&!%:;@~'`"* and the code unit 0x0022 (QUOTATION MARK).
           1. Let _toEscape_ be StringToCodePoints(_otherPunctuators_).
-          1. If _toEscape_ contains _c_ or _c_ is matched by |WhiteSpace| or |LineTerminator|, then
+          1. If _toEscape_ contains _c_, _c_ is matched by |WhiteSpace| or |LineTerminator|, or _c_ is in the inclusive interval from U+D800 to U+DFFF (i.e. _c_ is a leading surrogate or trailing surrogate), then
```

Comment:
```suggest
          1. If _toEscape_ contains _c_, _c_ is matched by |WhiteSpace| or |LineTerminator|, or _c_ is a leading surrogate or a trailing surrogate, then
```

Reply:
Strictly speaking, leading and trailing surrogates are defined to be code _units_, not code points. I figure it's close enough for the parenthetical, but not the actual algorithm step. I could change the parenthetical to be "_c_ is the code point corresponding to a leading surrogate or trailing surrogate" but that's getting pretty wordy.

Reply:
other than the oxford comma, this suggestion is a nit, so i'm ambivalent